.ci/setup.sh

```diff
@@ -4,7 +4,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-set -e
+set -ex
```

Comment:
addition of `-x` intentional?

Reply:
oh, good catch, it was for debugging.
Fixed, thx.